schema/private-influencers.yml

```diff
@@ -0,0 +1,127 @@
+private_influencers:
+  salutation:
+    - string
+    - optional
+    - salutation of the contact
+    - example: "Mr"
+  first_name:
+    - string
+    - required
+    - first name of the contact
+    - example: "John"
+  last_name:
+    - string
+    - required
+    - last name of the contact
+    - example: "Doe"
+  phonenumber_1:
+    - string
+    - optional
+    - phone number of the contact
+    - example: "123456789"
+  phonenumber_2:
+    - string
+    - optional
+    - phone number of the contact
+    - example: "123456789"
+  faxnumber:
+    - string
+    - optional
+    - fax number of the contact
+    - example: "123456789"
+  email_1:
+    - string
+    - required
+    - email of the contact
+    - example: "bill@example.com"
+  email_2:
+    - string
+    - optional
+    - email of the contact
+    - example: "bill@secondexample.com"
+  job_title:
+    - string
+    - optional
+    - job title of the contact
+    - example: "CEO"
+  language:
+    - string
+    - optional
+    - language of the contact
+    - example: "English"
+  twitter:
+    - string
+    - optional
+    - full URL of the twitter profile
+    - example: "https://twitter.com/BBCNews"
+  linkedin:
+    - string
+    - optional
+    - full URL of the linkedin profile
+    - example: "https://www.linkedin.com/company/bbc-news"
+  private_influencer_type:
+    - string
+    - optional
+    - the source system of the influencer
+    - example: "GORKANA | NGC3"
+  private_influencer_id:
+    - string
+    - optional
+    - id of the influencer on the source system
+    - example: "12345"
+  notes:
+    - string
+    - optional
+    - notes about the contact
+    - example: "This is a note"
+  line1:
+    - string
+    - optional
+    - address line 1
+    - example: "123 Fake Street"
+  line2:
+    - string
+    - optional
+    - address line 2
+    - example: "Fake Town"
+  line3:
+    - string
+    - optional
+    - address line 3
+    - example: "Fake County"
+  city:
+    - string
+    - optional
+    - city
+    - example: "Fake City"
+  state:
+    - string
+    - optional
+    - state
+    - example: "Fake State"
+  county:
+    - string
+    - optional
+    - county
+    - example: "Fake County"
+  postalcode_zipcode:
+    - string
+    - optional
+    - postal code or zip code
+    - example: "12345"
+  countryname:
+    - string
+    - optional
+    - country name
+    - example: "Fake Country"
+  private_influencer_outlet_type:
+    - string
```

Comment:
This seems like a new field . 
There will be only private contacts in these file as far as I know .
so what do we mean by **private_influencer_outlet_type** ? as there is already a field **private_influencer_type** .

Reply:
In the existing `sample.csv` there are two fields: `mdm_outlet_id` and `gorkana_outlet_id`. These fields are actually ignored in the actual import. I was not privy to the decision to include these fields but I expect it was to provide the ability to track the source of the information? The new schema just includes these fields as a generic update of these original fields.

I can't provide any more insight as to the use of these fields, and it might be possible to remove them?

Reply:
My understanding is that in NGC3, a private influencer can be linked to an Outlet, whether private or public. 

My assumption here is that `private_influencer_outlet_type` and ` private_influencer_outlet_id` serve as reference to the outlet which is linked to this private influencer. 

It sounds like CisionOne makes no use of this reference. Does this mean that CisionOne does not support linking a private contact to an outlet?